cli/chaosli/cmd/root.go

```diff
@@ -86,34 +88,56 @@ func initConfig() {
 }
 
 func DisruptionFromFile(path string) (v1beta1.Disruption, error) {
-	yaml, err := os.Open(filepath.Clean(path))
+	parsedSpec := ReadUnmarshalValidate(path)
+
+	return parsedSpec, nil
+}
+
+func DisruptionToFile(path string, spec v1beta1.DisruptionSpec) error {
+	return nil
+}
+
+func validatePath(filePath string) error {
```

Comment:
The only place this is used is to make sure the `path` flag is set for certain commands. cobra has a better way to do this. In the `init()` function of each command, you can do `contextCmd.MarkFlagRequired("path")` (or whatever the cmd variable is)